src/main/reactjs/src/tests/jest/__mocks__/globalMocks.ts

```diff
@@ -1,15 +1,8 @@
 import { jest } from '@jest/globals';
 
-jest.mock('react-i18next', () => ({
-  useTranslation: () => {
-    return {
-      t: (str: string) => str,
-    };
-  },
-}));
-
 jest.mock('configs/i18n', () => ({
   getCurrentLang: () => 'fi-FI',
   getSupportedLangs: () => ['fi-FI', 'sv-SE', 'en-GB'],
   changeLang: (language: string) => language,
+  useAppTranslation: () => ({ t: (str: string) => str }),
```

Comment:
Oikea `useAppTranslation` ottaa valinnaisen `keyPrefix`in. Olisiko testeissä kuitenkin parempi rendata snapshoteissa koko käännösavaimet eikä pelkästään loppuosaa?

Reply:
Ihan hyvä pointti! Itselläkään ei ole vahvaa kantaa. Tutkin hieman asiaa ja vaikuttaa siltä että suurimmassa osassa esimerkkeissä on käytetty  vain ns. loppuosaa. On tehty varmaan tällä tavalla, koska testejä kirjoittaessa ei silloin tarvitse katsoa lokalisaatio tiedostoja ollenkaan. Minun puolestani voidaan vielä mennä tällä tavalla ja siirtyä tarvittaessa käyttämään toista vaihtoehtoa.